### ISSUE TITLE
Record whether a veteran with a travel board hearing prefers to convert to a video or virtual hearing

### ISSUE THREAD

**Issue by tomas-nava** [department-of-veterans-affairs/caseflow#14898]
Connects #14857

## Description
Before we can schedule a video or virtual hearing for a veteran who had requested a travel board hearing, we must record which of those two types of hearing the veteran prefers, so that a coordinator can choose the requested type when scheduling.

Add a column to the `legacy_appeals` table named `changed_hearing_request_type`, which can have possible values of `nil`, `"video"`, or `"virtual"`.

`LegacyHearing.sanitized_hearing_request_type` should return `:video` or `:virtual` if the `changed_hearing_request_type` is not `nil`; otherwise it should return the (sanitized) hearing request type saved in VACOLS, as it does now.

Review how other objects and views use `LegacyAppeal.sanitized_hearing_request_type` and make sure they can handle a `:virtual` value. 

Review also how other objects and views use `LegacyAppeal.hearing_request_type`; should they use `sanitized_hearing_request_type` instead? If we've recorded a virtual or video hearing request type in Caseflow, will other objects behave inconsistently based on the value of `hearing_request_type`?

## Out of scope

In a future ticket, this request type will be displayed in Caseflow, to guide a coordinator when they're scheduling the hearing. Displaying the request type is out of scope for this ticket.

## Acceptance criteria
- [ ] A video or virtual hearing request type can be recorded for an individual hearing
